propTables.js

```diff
@@ -18,6 +18,7 @@ const removeProps = (component, props) => {
 	}
 
 	props.forEach(prop => {
+		// eslint-disable-next-line react/forbid-foreign-prop-types
```

Comment:
I think we could remove this since `npm run lint` doesn't catch this as a warning.